index.html

```diff
@@ -14,16 +14,18 @@
             <img src="img/Subtract.png" alt="img azul">
             <img src="img/Sports.png" alt="img sports">
         </div>
+        <img id="menuIcon" src="img/menu_hamburguesa.png" alt="menu">
         <nav id="navBar">
-            <ul id="navBarUl">
-                <li><a class="navLink" href="#">Sports</a></li>
-                <li><a class="navLink" href="#">Products</a></li>
-                <li><a class="navLink" href="#">About Us</a></li>
-                <li><a class="navLink" href="#">Training</a></li>
-                <li><a class="navLink" href="#">Community</a></li>
-            </ul>          
-        </nav>
-        <button>Contact</button>
+        <ul id="navBarUl">
```

Comment:
This list shouldn't be aligned with nav element, should be inside nav element. Also with the closing nav tag and the button tag

![image](https://github.com/nikollquinteroch/FrontEnd-Final-Evaluation/assets/136136120/24fa55c3-78fc-4b80-a92c-d3652e4ebac3)